CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ When reporting, please include any relevant details, links, screenshots, context
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers of this project pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of this project pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation. This project further recognizes oppression as directional and complex and will adjust its perspective on issues accordingly.
 
```

Comment:
I'm a tad late to the party, but wondering a bit about the `oppression is directional` part. I understand what it means, and I think I know what you're trying to get across by explicitly stating it, but I'm not sure if this is needed.